data/campaigns/The_South_Guard/scenarios/07b_Pebbles_in_the_Flood.cfg

```diff
@@ -156,6 +156,8 @@
 
         {MODIFY_UNIT x,y=24-40,5-25 facing sw}
 
+        {FREEZE_UNIT "Mal M'Brin"} # Prevents player from pinning unit away from keep
+
```

Comment:
The macro is rather unnecessary; the [object] could just be given to him in the side definition instead. Secondly, you don't need to list all terrains; the castle is surrounded only by flat and mountains, so those two (or three if you want to include castle) would suffice. Thirdly, the usual impassable cost is 99 (or, {UNREACHABLE}), which would be good to use simply for greppability's sake. Fourthly, one might count the fact that the player sees the movement limitation as a small fault (for an event-based solution, see for example commit 6d8b4202), but in this case it's probably not a big deal.

I now noticed the macro is from DM, to which it probably got from SotA, and it'd be good to change that 99/100 thing in those places too.

Reply:
Thanks for pointing that out, I'll take care of those.